src/services/item/plugins/geolocation/service.ts

```diff
@@ -43,15 +43,22 @@ export class ItemGeolocationService {
     actor: Actor,
     repositories: Repositories,
     query: {
-      lat1: ItemGeolocation['lat'];
-      lat2: ItemGeolocation['lat'];
-      lng1: ItemGeolocation['lng'];
-      lng2: ItemGeolocation['lng'];
+      parentItemId?: Item['id'];
```

Comment:
Maybe it could be interesting to do the same as in the repository ? Trying to enforce the mandatory of types.